src/popup/router/index.ts

```diff
@@ -69,7 +69,11 @@ router.beforeEach(async (to, from, next) => {
     return;
   }
 
-  if (!store.getters['sdkPlugin/sdk'] && !RUNNING_IN_POPUP) initSdk();
+  const { isSdkReady } = useSdk13({ store });
+
+  if (!isSdkReady.value && !RUNNING_IN_POPUP) {
+    initSdk();
+  }
```

Comment:
I think we should remove this as the SDK is not needed on every route. Especially that the SDK instance is lazy loaded when `getSdk` method is used.

Reply:
IDEM https://github.com/aeternity/superhero-wallet/pull/2164#discussion_r1266318401